kitsune/users/tests/test_views.py

```diff
@@ -121,6 +120,18 @@ def test_deactivate_and_flag_spam(self):
         eq_(1, Answer.objects.filter(creator=u, is_spam=True).count())
         eq_(0, Answer.objects.filter(creator=u, is_spam=False).count())
 
+    def test_inbox_deletion_on_deactivation(self):
+        """Test message deletion on deactivation"""
+        recipient = UserFactory()
+        sender = UserFactory()
+        InboxMessage.objects.create(
+            to=recipient, sender=sender, message="testing message deletion"
+        )
```

Comment:
Can we use a factory here, or even better, log in as the sender and create the message through a post request. That way we can also test that we're not deleting messages from the sender's outbox.